pulp/tests/test_pulp.py

```diff
@@ -787,6 +787,24 @@ def test_timeLimit(self):
         # CHOCO has issues when given a time limit
         if self.solver.name != 'PULP_CHOCO_CMD':
             pulpTestCheck(prob, self.solver, [const.LpStatusOptimal], {x: 4, y: -1, z: 6, w: 0})
+    
+    def test_assignInvalidStatus(self):
+        print("\t Testing invalid status")
+        t = LpProblem('test')
+        Invalid = -100
+        try:
```

Comment:
One last thing, could you use the `self.assertRaises` here? I think is cleaner. Check this recent example: https://github.com/coin-or/pulp/blob/3414dc943e2985e0b20158a0850b9771e62fd9b6/pulp/tests/test_pulp.py#L846

Reply:
Thanks for introducing this method to me, the test has been modified.